package-lock.json

```diff
@@ -2714,6 +2714,14 @@
       "dev": true,
       "requires": {
         "lodash": "4.17.11"
+      },
+      "dependencies": {
+        "lodash": {
+          "version": "4.17.11",
```

Comment:
Should this have been bumped as well?

Reply:
is that a direct dependency or a sub dependency of another package?

Reply:
Either way, snyk seems happy, so I see this as an absolute win.

Reply:
Ironically this is in a `snyk` dependency... so hopefully they'll update it soon: `@snyk/composer-lockfile-parser`